src/c_kzg_4844.c

```diff
@@ -825,10 +825,12 @@ void free_trusted_setup(KZGSettings *s) {
   free_kzg_settings(s);
 }
 
-static void compute_powers(fr_t out[], uint64_t n) {
-    out[0] = fr_one;
-    for (uint64_t i = 2; i < n; i++)
-      fr_mul(&out[i], &out[i-1], &out[1]);
+static void compute_powers(fr_t out[], BLSFieldElement *x, uint64_t n) {
```

Comment:
It's probably better for readability to call them all `fr_t` or all `BLSFieldElement` rather than mixing them.

Reply:
Done in bcf14e7